frontend/src/app/filter/boolean-filter/boolean-filter.component.html

```diff
@@ -1,4 +1,11 @@
 <div class="bool" *ngIf="filter">
-    <p-checkbox [ngModel]="data" binary="true" (onChange)="update($event.checked)"></p-checkbox>
-    {{data | json | titlecase }}
+    <ia-dropdown [value]="data" (onChange)="update($event)"
+        labelledBy="legend-filter-{{filter.displayName | slugify}}">
+        <span iaDropdownLabel>{{data !== undefined ? (data | json) : 'any value'}}</span>
+        <div iaDropdownMenu>
+            <a iaDropdownItem>any value</a>
```

Comment:
Not big on this label. It obfuscates the fact that the filter is a boolean. But also hard pressed to find a better alternative.

`True or False`? (might get into arguments with logicians with this kind of wording..)

Reply:
Hm, true, the label is not great.

Re. "true or false": the default state of the filter allows `{ true, false, null }`, not `{ true, false }`. So logic pedantry aside, "true or false" does not really reflect the value, if the field is empty for some documents. Though to be fair, the label "any value" also does not really reflect that this includes "no value".

(Also: yes, this makes the distinction between booleans and keyword fields less clear, but for the user, that distinction is perhaps not that important?)